purchases/src/androidTestDefaults/kotlin/com/revenuecat/purchases/offlineentitlements/OfflineEntitlementsWithInitialRequestsCompletedIntegrationTest.kt

```diff
@@ -288,7 +289,7 @@ class OfflineEntitlementsWithInitialRequestsCompletedAndNoInitialPurchasesIntegr
 
                     Purchases.resetSingleton()
                     mockActivePurchases(initialActivePurchases)
-                    configureSdk(activity, forceServerErrors = false)
+                    Purchases.configureSdk(activity, testUserId, mockBillingAbstract, forceServerErrors = false)
```

Comment:
I believe this was broken in a bad rebase. It was causing the integration tests to not compile 🤦. They did run correctly in the 6.8.0 release

Reply:
Weird, but they passed?

Reply:
At that point yes, seems that I broke it in https://github.com/RevenueCat/purchases-android/pull/1164. They did run locally fine for me but I think I might have broken it on a rebase.